<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-21</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="continueBotButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>200</y>
        <width>281</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>PLAYER VS BOT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>50</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>35</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Warships</string>
      </property>
     </widget>
     <widget class="QPushButton" name="continueNetButton">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>340</y>
        <width>281</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>PLAYER VS PLAYER</string>
      </property>
     </widget>
     <widget class="QFrame" name="framePort">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>460</y>
        <width>211</width>
        <height>151</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPlainTextEdit" name="portInput">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>104</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="submitNetButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Submit Button</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="hostCheckBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>78</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Your field</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enemy field</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>190</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLabel" name="readyLabel">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>140</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>You are ready</string>
      </property>
     </widget>
     <widget class="QLabel" name="NotReadyLabel">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>140</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>You are not ready</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
